libs/blocks/global-navigation/global-navigation.js

```diff
@@ -822,6 +822,7 @@ class Gnav {
         ${isDesktop.matches ? '' : this.decorateSearch()}
         ${this.elements.mainNav}
         ${isDesktop.matches ? this.decorateSearch() : ''}
+        ${getConfig().searchEnabled === 'on' ? toFragment`<div class="feds-client-search"></div>` : ''}
```

Comment:
One thing is not very clear to me. This feature is supposed to be relevant for the standalone Gnav, yet the config is added through `scripts.js` at a consumer level. Couldn't a config be passed directly to the standalone version? I feel like I'm missing something, but also want to make sure we're not polluting `scripts.js` with irrelevant properties.

Reply:
@overmyheadandbody There is no separate config for standalone Gnav. The standalone version directly calls the init of global-navigation /global-footer and we set the miloLibs from `navigation.js` not through script.js.